doc/rg.1.txt.tpl

```diff
@@ -82,7 +82,7 @@ _PATH_::
 
 OPTIONS
 -------
-Note that for many options, there exist flags to disable them. In some cases,
+Note that many options can be disabled through flags. In some cases,
```

Comment:
`with`/`via` flags?

Reply:
"[W]ith flags" and "via flags" are also acceptable.  "[V]ia flags" is probably the best choice.